chemprop/train/train.py

```diff
@@ -50,65 +52,75 @@ def train(model: MoleculeModel,
             batch.atom_features(), batch.bond_features(), batch.data_weights()
 
         mask = torch.tensor(mask_batch, dtype=torch.bool) # shape(batch, tasks)
-        targets = torch.tensor([[0 if x is None else x for x in tb] for tb in target_batch]) # shape(batch, tasks)
+        targets = torch.tensor([[0 if x is None else x for x in tb] for tb in target_batch])  # shape(batch, tasks)
 
         if args.target_weights is not None:
             target_weights = torch.tensor(args.target_weights).unsqueeze(0) # shape(1,tasks)
         else:
             target_weights = torch.ones(targets.shape[1]).unsqueeze(0)
-        data_weights = torch.tensor(data_weights_batch).unsqueeze(1) # shape(batch,1)
+        data_weights = torch.tensor(data_weights_batch).unsqueeze(1)  # shape(batch,1)
 
-        if args.loss_function == 'bounded_mse':
-            lt_target_batch = batch.lt_targets() # shape(batch, tasks)
-            gt_target_batch = batch.gt_targets() # shape(batch, tasks)
+        if args.loss_function == "bounded_mse":
+            lt_target_batch = batch.lt_targets()  # shape(batch, tasks)
+            gt_target_batch = batch.gt_targets()  # shape(batch, tasks)
             lt_target_batch = torch.tensor(lt_target_batch)
             gt_target_batch = torch.tensor(gt_target_batch)
 
         # Run model
         model.zero_grad()
-        preds = model(mol_batch, features_batch, atom_descriptors_batch, atom_features_batch, bond_features_batch)
+        preds = model(
+            mol_batch,
+            features_batch,
+            atom_descriptors_batch,
+            atom_features_batch,
+            bond_features_batch,
+        )
 
         # Move tensors to correct device
         torch_device = preds.device
         mask = mask.to(torch_device)
         targets = targets.to(torch_device)
         target_weights = target_weights.to(torch_device)
         data_weights = data_weights.to(torch_device)
-        if args.loss_function == 'bounded_mse':
+        if args.loss_function == "bounded_mse":
             lt_target_batch = lt_target_batch.to(torch_device)
             gt_target_batch = gt_target_batch.to(torch_device)
 
         # Calculate losses
-        if args.loss_function == 'mcc' and args.dataset_type == 'classification':
-            loss = loss_func(preds, targets, data_weights, mask) *target_weights.squeeze(0)
-        elif args.loss_function == 'mcc': # multiclass dataset type
+        if args.loss_function == "mcc" and args.dataset_type == "classification":
+            loss = loss_func(preds, targets, data_weights, mask) * target_weights.squeeze(0) * mask.sum()
```

Comment:
why multiply by `mask.sum()`? The MCC is already a batch-level value by its nature, so we shouldn't be modifying this value by the size of the batch. I think this might've gotten lost in my initial comment.

Reply:
Whoops sorry left this in by accident. Lost track of it when I updated the formatting.